test/commands/scanner/run.severity.test.ts

```diff
@@ -128,7 +129,8 @@ describe('scanner:run', function () {
 					'--format', 'json'
 				])
 				.it('Ensure normalized severity is not outputted when --normalize-severity not provided', ctx => {
-					const output = JSON.parse(ctx.stdout);
+					// The first line is a header. Strip that away, and parse the rest as JSON.
+					const output = JSON.parse(ctx.stdout.split('\n')[1]);
```

Comment:
Can we move all the banner-related test fixes to a smaller area (such as a separate module with functions that can be invoked) so that there'll be less code to clean up once we remove the banner?

Reply:
@jfeingold35 @rmohan20 I have the same comments as PR 748 and PR 752. Can we discuss Monday afternoon via huddle?

So I would consider this message a toast, so let's follow CCX toast conventions.

My three suggestions:

"We released a new version of Salesforce Code Analyzer. Tell us what you think! Give Feedback."

OR

"We've improved Salesforce Code Analyzer. Tell us what you think! Give Feedback."

OR

"We're constantly improving Salesforce Code Analyzer. Tell us what you think! Give Feedback."

Link on Give Feedback-Can we include a Give Feedback button? To see more of what I'm talking about, check out the "spotlight on a toast" about 25% down this page: https://confluence.internal.salesforce.com/display/DOCTEAM/toast